.github/workflows/wf-test-golib.yml

```diff
@@ -23,7 +23,6 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
           cache: ${{ inputs.go-cache }}
-          working-directory: ${{ inputs.working-directory }}
```

Comment:
`working-directory` seems exist in this file at 
https://github.com/mackerelio/opentelemetry-collector-mackerel/blob/5a94d8118f4ee584840d7cb8436b32ef8889e7f2/.github/workflows/wf-test-golib.yml#L9-L11

Reply:
thanks. i understand.